core/src/impl/Kokkos_SimpleTaskScheduler.hpp

```diff
@@ -221,7 +220,7 @@ class SimpleTaskScheduler
     // SharedAllocationRecord pattern
     using record_type =
         Impl::SharedAllocationRecord<memory_space,
-                                     Impl::DefaultDestroy<task_queue_type> >;
+                                     Impl::DefaultDestroy<task_queue_type>>;
```

Comment:
I find it a bit interesting that this wasn't caught by clang-format check before.